core/api/src/app/merchants/approve-merchant-map.ts

```diff
@@ -0,0 +1,13 @@
+import { MerchantsRepository } from "@/services/mongoose"
+
+export const approveMerchantById = async (
+  id: MerchantId,
+): Promise<BusinessMapMarker | ApplicationError> => {
+  const merchantsRepo = MerchantsRepository()
+
+  const merchant = await merchantsRepo.findById(id)
+  if (merchant instanceof Error) return merchant
+
+  merchant.validated = true
```

Comment:
Can be a method on the merchants repo similar to how we have things like `markAsPaid` and `markAsProcessingCompleted`